locales/en-US/app.ftl

```diff
@@ -905,6 +905,19 @@ TrackPower--tooltip-energy-carbon-used-in-preview-milliwatthour = { $value } mWh
 TrackPower--tooltip-energy-carbon-used-in-preview-microwatthour = { $value } µWh ({ $carbonValue } mg CO₂e)
   .label = Energy used in the current selection
 
+## TrackBandwidth
+
+TrackBandwidthGraph--speed = { $value } per second
```

Comment:
Please add comments for all these variables, similarly to what it's done in the lines above.

Note that you can use group-level comments where possible, to apply a comment to more than one string.
https://projectfluent.org/fluent/guide/comments.html